code-green/workshop/README.md

```diff
@@ -66,6 +66,24 @@ When you're done, this is what you'll have created:
 	title="Workshop architecture diagram" style="width:90%" />
 <br><br>
 
+#### Shortcut to the End
+
+If you encounter issues or run short on time and want to jump to the end, we have you covered.
```

Comment:
Can you add a little more explanation to make this more clear for beginners--that it's a script to automatically create everything that's done manually if you go through the workshop yourself?